src/WebJobs.Script.WebHost/Models/HostAssignmentContext.cs

```diff
@@ -129,6 +129,23 @@ public void ApplyAppSettings(IEnvironment environment)
                     environment.SetEnvironmentVariable(EnvironmentSettingNames.CorsAllowedOrigins, allowedOrigins);
                 }
             }
+
+            if (EasyAuthSettings != null)
+            {
+                // App settings take precedence over site config for easy auth enabled.
+                if (string.IsNullOrEmpty(environment.GetEnvironmentVariable(EnvironmentSettingNames.EasyAuthEnabled)))
```

Comment:
i dont think this wil ever be false. Is there some specific scenario this check is handling?